featuretools/primitives/base/aggregation_primitive_base.py

```diff
@@ -109,14 +110,16 @@ def make_agg_primitive(function, input_types, return_type, name=None,
         base_of_exclude (list[PrimitiveBase]): Blacklist of
             primitives that cannot have this primitive in input_types.
 
-        description (str): Description of primitive.
```

Comment:
let's keep description in as a default value

Reply:
> let's keep description in as a potential argument

Fixed. Now description will be fixed in this order:

1 - description sent through argument. 
2 - if 1 is None: description will be the function's docstring.
3 - if 1 and 2 are None: description will be a default description.